src/main/java/com/wcc/platform/service/MentorshipService.java

```diff
@@ -80,6 +82,24 @@ public MentorshipCodeOfConductPage getCodeOfConduct() {
   }
 
   /**
+
+   * API to retrieve information about the study groups.
+   *
+   * @return Mentorship study groups page.
+   */
+  public MentorshipStudyGroupPage getStudyGroups() {
+    final var page = repository.findById(STUDY_GROUPS.getId());
+    if (page.isPresent()) {
+      try {
+        return objectMapper.convertValue(page.get(), MentorshipStudyGroupPage.class);
+         } catch (IllegalArgumentException e) {
+        throw new PlatformInternalException(e.getMessage(), e);
+      }
+    }
+      return repository.getFallback(STUDY_GROUPS, MentorshipStudyGroupPage.class, objectMapper);
```

Comment:
Also please have a look in the format it is not formatted correctly. But intelliJ is working properly, just re-format by changing any line and this will be fixed.